skip musttail on armv6:

fatal error: error in backend: failed to perform tail call elimination on a call site marked musttail

diff --git a/Source/WTF/wtf/Compiler.h b/Source/WTF/wtf/Compiler.h
index 0ea5cb7..3bf5478 100644
--- a/Source/WTF/wtf/Compiler.h
+++ b/Source/WTF/wtf/Compiler.h
@@ -284,7 +284,7 @@
 /* MUST_TAIL_CALL */
 
 #if !defined(MUST_TAIL_CALL) && defined(__cplusplus) && defined(__has_cpp_attribute)
-#if __has_cpp_attribute(clang::musttail)
+#if __has_cpp_attribute(clang::musttail) && !(defined(__ARM_ARCH) && __ARM_ARCH == 6)
 #define MUST_TAIL_CALL [[clang::musttail]]
 #endif
 #endif
